### DISCUSSION TITLE
iOS 14+ About Limited PhotoLibrary

### DISCUSSION THREAD

**Discussion by Ich-bin-naoya** [react-native-image-picker/react-native-image-picker#1564] [General]
I tried to set a "limited permission setting" for photos. So only the selected photos can be shown on the app. 
However, when I ran the launchImageLibrary, all the photos (including the ones not permitted) are shown.

Is there any other way to show the photos that the user only selected?

I use [ react-native-permissions](https://github.com/zoontek/react-native-permissions) to check permission

**Comment by makedirectory** [comment]
Same issue here

**Comment by ravirajn22** [comment]
This library does not require permission (provided iOS version >= 11, thats how native picker works) for selecting asset from photos library. So no permission is applicable.

**Reply by gwmccull** [reply]
per this article, this library should be switching from `UIImagePickerController` to `PHPicker` in order to respect and use the limited photo library access

https://www.andyibanez.com/posts/understanding-limited-photo-library-ios-14/

**Reply by Johan-dutoit** [reply]
Latest version supports PHPicker (the release is currently in progress)

**Reply by gwmccull** [reply]
❤️ thanks for your hard work on this library!

**Reply by jjjeeerrr111** [reply]
Any news on `PHPicker`? Is this available now?

**Reply by FadiAboMsalam** [reply]
@ravirajn22 any update on this ?

**Comment by thatgriffith** [comment]
I also have the same issue. Any news on this?

**Reply by makedirectory** [reply]
You can use this for iOS 14+

https://github.com/makedirectory/react-native-photo-picker

**Reply by thatgriffith** [reply]
> You can use this for iOS 14+
> 
> 
> 
> https://github.com/makedirectory/react-native-photo-picker

Thanks. Will try it out!
